org.mwc.debrief.pepys/src/main/java/org/mwc/debrief/pepys/model/ModelConfiguration.java

```diff
@@ -92,7 +100,7 @@ public void apply() throws NoSuchMethodException, SecurityException, Instantiati
 	public void doImport() {
 		if (_bridge == null) {
 			doImportProcessMockup(treeModel);
-		}else {
+		} else {
 			doImport(treeModel, new InternTreeItemFiltering() {
```

Comment:
@saulhidalgoaular  - I could do with a comment explaining what's happening with these two `doImport()` statements. I guess it's to ensure `States` are loaded before `Contacts` - but we should describe that, please.